feat(curriculum): add steps for employee profile generator workshop (#64896)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-employee-profile-generator/695c96f82af4fb78e59fa9ed.md

---
id: 695c96f82af4fb78e59fa9ed
title: Step 1
challengeType: 20
dashedName: step-1
---

# --description--

> Strings are sequence of characters used to store text data. As you might recall from previous lessons, you can create a string by enclosing text inside either single('') or double("") quotes. For example:

Reviewer: ```suggestion
Strings are sequences of characters used to store text data. As you might recall from previous lessons, you can create a string by enclosing text inside either single (`'`) or double (`"`) quotes. For example:
```

```python
greeting = 'Hello'
print(greeting) # Output: Hello
```

Create a variable `first_name` which stores the string `John` and a variable `last_name` which stores the string `Doe`. Then print `first_name` and `last_name`.

# --hints--

You should have a variable named `first_name`.

You should assign the string `John` to your `first_name` variable.

```js
({ test: () => true })
```

You should have a variable named `last_name`.

> 

Reviewer: You still need the test after the hint.
````suggestion

```js
({ test: () => true })
```

You should assign the string `John` to your `first_name` variable.

```js
({ test: () => true })
```

You should have a variable named `last_name`.

```js
({ test: () => true })
```

````

You should assign the string `Doe` to your `last_name` variable.

```js
({ test: () => true })
```

Print the `first_name` variable.

```js
({ test: () => true })
```

> Print the `last_name` variable.

Reviewer: ```suggestion
You should print your `last_name` variable to the console.
```

---
id: 695c96f82af4fb78e59fa9ed
title: Step 1
challengeType: 20
dashedName: step-1
---

# --description--

Strings are sequences of characters used to store text data. As you might recall from previous lessons, you can create a string by enclosing text inside either single (`'`) or double (`"`) quotes. For example:

```python
greeting = 'Hello'
print(greeting) # Output: Hello
```

Create a variable `first_name` which stores the string `John` and a variable `last_name` which stores the string `Doe`. Then print `first_name` and `last_name`.

# --hints--

You should have a variable named `first_name`.

```js
({ test: () => true })
```

You should assign the string `John` to your `first_name` variable.

```js
({ test: () => true })
```

You should have a variable named `last_name`.

```js  
({ test: () => true })   
```

You should assign the string `Doe` to your `last_name` variable.

```js
({ test: () => true })
```

You should print your `first_name` variable to the console. 

```js
({ test: () => true })
```

> You should print your `last_name` variable to the console. 

Reviewer: ```suggestion
You should print your `last_name` variable to the terminal. 
```
